### ISSUE TITLE
Can I update this mod to v7+?

### ISSUE THREAD

**Issue by ThomasThings** [Garmundo/Extended-unit-arsenal#16]
I saw Lixie (Blu, Bluewolf, Allure/Exotic Mod Developer) do a thing where she resprited an old mod and redid it,

I was wondering if I could do that with yours. I'd be redoing all of the sprites, keeping their shapes and stuff faithful, and I'd be recreating the code from scratch in java.

**Comment by Garmundo** [comment]
YES! Please do! I'm sure there's people in the community who miss this mod.
I never got around to updating after the massive overhaul of v6 and my
attention has been on other things.

This mod was a little side thing I did when school was out because of...
2020... since modding this game was made so easy. Shared it and I think a
lot of people liked it so I hope you can bring back the charm for newer and
older players alike and make it even better! Best of luck! Maybe I'll come
back to this game sometime but I don't know.

**Comment by Garmundo** [comment]
If any of the sprites are still up to standard or if the code still works;
I won't mind if you use them if it helps the process.
